### PR TITLE
update to proto comments for documentation

### PR DESCRIPTION
**Pull request by joshLong145**
Update to protobuf comments for schemas
## Changes

- Item 1
- Item 2

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: proto/motor/v1/request.proto [hunk 1]
```diff
@@ -106,24 +105,40 @@ message QueryWhoIsRequest {
...
   string creator = 1;
+  // did (identifier) of the specific schema
   string did = 2;
 }
 
+// Query for all WhatIs instances for a single creator address (snr address)
 message QueryWhatIsByCreatorRequest {
+  // Address to query on
   string creator = 1;
+
```
Comment (mcjcloud) [inline proto/motor/v1/request.proto:161]: no comment for this one?

**Review by mcjcloud — CHANGES_REQUESTED** [review]
Looks like the spacing between properties is inconsistent; with some having a newline between properties and others not.

File: proto/motor/v1/request.proto (hunk 1 above)
Comment (joshLong145) [reply proto/motor/v1/request.proto:161]: will add

**Review by ntindle — APPROVED** [review]
Much better

**Review by mcjcloud — APPROVED** [review]

**Comment by joshLong145** [comment]
Will be migrating this to the #655 branch

**Review by prdlk — APPROVED** [review]

**Pull request merged**